resourcepolicies.md

```diff
@@ -0,0 +1,45 @@
+# Resource Policies Endpoints
+[Back to the list of all defined endpoints](endpoints.md)
+
+## Main Endpoint
+**/api/authz/resourcepolicies**   
+
+As we don't have yet an use case to iterate over all the existent resource policies the main endpoint is not implemented and a 405 error code is returned according to our [general error response codes](README.md#Error codes).
+
+Example: <http://dspace7.4science.it/dspace-spring-rest/#/dspace-spring-rest/api/core/collections>
```

Comment:
The example URL is incorrect.

Reply:
Oh thanks. I removed the example url as it is related to a wrong call and it is already shown in the header of the paragraph (so no reason to provide additional suggestion :) ). Moreover the endpoint is not implemented in the master